parser/proto.y

```diff
@@ -721,34 +678,58 @@ tagRanges : tagRange {
 		$$ = &rangeList{$1, $2, $3}
 	}
 
-tagRange : _INT_LIT {
+tagRange : tagRangeStart {
```

Comment:
I think this was actually clearer before. Let's undo the changes and keep the three cases inlined. While they help with readability of the grammar in the language spec, the explosion to many productions hurts readability here IMO.

So I think you could just revert all of the changes to these ranges (and remove the `rangeEnd` type added to `ast.go`) and instead only do renames: `enumRange`, `enumRanges`, and `intLit` to `enumValueRange`, `enumValueRanges`, and `enumValueNumber` respectively.

Reply:
I reverted the changes here and just do the renaming.